src/ui/win/win_ui.cpp

```diff
@@ -66,6 +66,19 @@ namespace WindowsUi
                 SelectObject( hdc, hOldPen );
                 DeleteObject( hPen );
             }
+
+            for ( std::vector < UiWindowWin::EllipseToDraw >::const_iterator ellipse = _window->_ellipse.cbegin(); ellipse != _window->_ellipse.cend(); ++ellipse ) {
+                const int _xTopLeft = static_cast<int>( ellipse->xTopLeft * xFactor);
```

Comment:
No need to add **_** for local variables. We do this only for members of a class.